.github/workflows/ci.yaml

```diff
@@ -0,0 +1,112 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
+
+        steps:
+            -
+                name: "Checkout code"
+                uses: "actions/checkout@v2"
+
+            -
+                name: "Install PHP"
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php-version }}"
+                    tools: "composer:v2"
+
+            -
+                name: "Validate composer.json"
+                run: "composer validate --strict --no-check-lock"
+
+            -
+                name: "Composer install"
+                uses: "ramsey/composer-install@v1"
+                with:
+                    composer-options: "--no-suggest --no-scripts"
+
+    tests:
+        needs:
+            - "composer-validate"
+
+        name: "PHP ${{ matrix.php-version }} + ${{ matrix.dependencies }} + Composer v${{ matrix.composer-version }}"
+
+        runs-on: ubuntu-16.04
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                dependencies:
+                    - 'lowest'
+                    - 'highest'
+                composer-version:
+                    - '1'
+                    - '2'
+                allow-failures: [false]
+
+        steps:
+            - name: "Checkout code"
+              uses: actions/checkout@v2.3.3
+
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v${{ matrix.composer-version }}
+
+            - name: "Add PHPUnit matcher"
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            - if: matrix.php-version == '8.0'
+              run: composer config platform.php 7.4.99 || true
+
+            - if: matrix.composer-version == '1'
+              run: composer require --no-update composer/composer:^1.0.2
+
+            - if: matrix.composer-version == '2'
+              run: composer require --no-update composer/composer:^2
```

Comment:
Can we remove these two steps, as we are now setting up the correct composer version three steps above? 🤔

Reply:
not sure what you mean here
this is about putting composer/composer in the vendor/ folder, it's unrelated to which version of composer is used to install the deps

Reply:
I mean we can remove these steps if we install the vendors directly using composer 1 or composer 2, right?
In this case the plugin api version is 1 or 2

Reply:
`composer-plugin-api": "^1.0|^2.0"` so this cannot be a selector

Reply:
ok but you want to test it in one workflow job instead, right?

Reply:
I don't know what this means :)
Same as in travis basically :)